app/plugin_install.go

```diff
@@ -177,6 +196,7 @@ func (a *App) removePlugin(id string) *model.AppError {
 		return model.NewAppError("removePlugin", "app.plugin.remove_bundle.app_error", nil, err.Error(), http.StatusInternalServerError)
 	}
 
+	// Notify cluster peers async.
```

Comment:
Is this comment adding value?

Reply:
just so the reader knows that this is an async call vs sync as we have both cases.